packages/js/examples/restart-ice/index.html

```diff
@@ -0,0 +1,463 @@
+        var client;
+        var currentCall = null;
+        var project = localStorage.getItem('relay.example.project') || '';
+        var token = localStorage.getItem('relay.example.token') || '';
+        var number = localStorage.getItem('relay.example.number') || '';
+        var numberFrom = localStorage.getItem('relay.example.numberFrom') || '';
+        var audio = localStorage.getItem('relay.example.audio') || '1';
+        var video = localStorage.getItem('relay.example.video') || '1';
+        var space = localStorage.getItem('relay.example.space') || '';
+        var apiToken = localStorage.getItem('relay.example.apiToken') || '';
+        var expiresIn = localStorage.getItem('relay.example.expiresIn') || '3600';
+        var resource = localStorage.getItem('relay.example.resource') || 'test-client';
+        const getCircularReplacer = () => {
+            const seen = new WeakSet();
+            return (key, value) => {
+                if (typeof value === "object" && value !== null) {
+                    if (seen.has(value)) {
+                        return;
+                    }
+                    seen.add(value);
+                }
+                return value;
+            };
+        };
+        /**
+         * On document ready auto-fill the input values from the localStorage.
+         */
+        ready(function() {
+            document.getElementById('space').value = space;
+            document.getElementById('project').value = project;
+            document.getElementById('apiToken').value = apiToken;
+            document.getElementById('expiresIn').value = expiresIn;
+            document.getElementById('resource').value = resource;
+            document.getElementById('token').value = token;
+            document.getElementById('number').value = number;
+            document.getElementById('numberFrom').value = numberFrom;
+            document.getElementById('audio').checked = audio === '1';
+            document.getElementById('video').checked = video === '1';
+        });
+        /**
+         * Generate JWT token using fetch
+         */
+        async function generateToken() {
+            let spaceVal = document.getElementById('space').value.trim();
+            const projectVal = document.getElementById('project').value.trim();
+            const apiTokenVal = document.getElementById('apiToken').value.trim();
+            const expiresInVal = document.getElementById('expiresIn').value;
+            const resourceVal = document.getElementById('resource').value.trim();
+
+            if (!spaceVal || !projectVal || !apiTokenVal) {
+                alert('Please fill in Space, Project ID, and API Token.');
+                return;
+            }
+
+            // Clean spaceVal if it has https:// or .signalwire.com
+            spaceVal = spaceVal.replace(/^https?:\/\//, '').replace(/\.signalwire\.com$/, '');
+
+            const url = `https://${spaceVal}.signalwire.com/api/relay/rest/jwt`;
```

Comment:
This will make it work for Production only. It's basically fine, but we should consider the option to define an entire domain that's not a `.signalwire.com` one.